Florida_Analysis_Py_File.py

```diff
@@ -0,0 +1,534 @@
+# Florida Data Analysis - Opioid Shipment And Overdosage Related Mortality
+
+## Pre-Post Analysis Of Opioid Shipments In Florida
+
+# importing the required libraries
+import pandas as pd
+import numpy as np
+import altair as alt
+import statsmodels.api as sm
+import statsmodels.formula.api as smf
+
+# loading data for drug prescriptions in Florida and neighbouring states
+prescriptions = pd.read_csv(
+    r"C:\Users\annap\OneDrive\Desktop\Opioid Project\Prescriptions.csv",
```

Comment:
Hey Annapurna, this is the updated comments for your analysis. Alex has re-uploaded datasets and removed some NA values. You might want to shift to that dataset and use MME in the formula directly to calculate the Opioid shipment.

Reply:
Thanks Zhonglin for these updated comments. I will make sure to work on them.